### PR TITLE
Fixes some cargo gun descriptions and lowers 38 ammo price.

### PR DESCRIPTION
**Pull request by Martinpachu**
## About The Pull Request

As the name says.

## Why It's Good For The Game

38 Special is supposed to be the cheapest ammo, doing the same damage as 9mm rounds and has guns either with low ammo capacity, or that need to be pumped after every shot. Also i fucked some things up in my last PR. sorry. sorry. im so stupid. dont hit me with the tail.

## Changelog

:cl:
balance: .38 ammo boxes are cheaper.
spellcheck: Fixed some cargo gun descriptions.
fix: Fixed an issue regarding multiple stechkin pistols in a singular cargo pack.
:cl:

### CODE REVIEW

**Comment by Martinpachu** [comment]
conflict's fixed!

File: code/modules/cargo/packs/gun.dm [hunk 1]
```diff
@@ -89,39 +89,39 @@
 
 /datum/supply_pack/gun/hellfire_shotgun
 	name = "Hellfire Shotgun Crate"
-	desc = "For when you need to deal with 7 hooligans. Contains a pump shotguns, with a 8-round capacity."
+	desc = "For when you need to deal with 7 hooligans. Contains a pump shotgun, with a 8-round capacity."
```
Comment (MarkSuckerberg) [inline code/modules/cargo/packs/gun.dm]:
```suggestion
	desc = "For when you need to deal with 8 hooligans. Contains a pump shotgun, with a 8-round capacity."
```

**Review by MarkSuckerberg — COMMENTED** [review]
just this one thing

File: code/modules/cargo/packs/gun.dm [hunk 2]
```diff
@@ -89,39 +89,39 @@
...
-	desc = "Contains a .45 submachine guns, manufactured by Scaraborough Arms and chambered in .45"
+	desc = "Contains a .45 submachine gun, manufactured by Scaraborough Arms and chambered in .45"
 	cost = 3000
 	contains = list(/obj/item/gun/ballistic/automatic/smg/c20r/cobra)
 	crate_name = "SMG crate"
 
 /datum/supply_pack/gun/illestren
 	name = "Illestren Rifle Crate"
-	desc = "Contains a expertly made bolt action rifles intended for hunting wildlife. Chambered in 8x50mmR rounds."
+	desc = "Contains a expertly made bolt action rifle intended for hunting wildlife. Chambered in 8x50mmR rounds."
```
Comment (SomeguyManperson) [inline code/modules/cargo/packs/gun.dm]:
```suggestion
	desc = "Contains an expertly made bolt action rifle intended for hunting wildlife. Chambered in 8x50mmR rounds."
```

**Comment by Martinpachu** [comment]
fixed this finally

**Comment by Martinpachu** [comment]
fixed...again.. fuck!!

**Pull request merged**